### ISSUE TITLE
Mini player is wrongly placed underneath the browser window in fullscreen mode (Ubuntu)

### ISSUE THREAD

**Issue by SoftVision-EmilPasca** [meandavejustice/min-vid#502]
[Affected versions]:
- Firefox 45.0 and up
- Min-Vid 0.3.3-dev

[Affected Platforms]:
- All Linux

[Prerequisites]:
- Latest Min-Vid add-on version (0.3.3-dev) was previously installed on a clean profile.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Navigate to www.youtube.com page.
3. Send a video to the mini player using the thumbnail overlay or context menu.
4. Press the "F11" key for fullscreen mode.
5. Observe the mini player behavior.

[Expected result]:
- Mini player is placed on top of the browser window.

[Actual result]:
- Mini player is hidden and can no longer be brought on top.

[Additional Notes]:
- This also occurs if you're already in fullscreen mode and you are sending a video to the mini player.
- Attached a screen recording of the issue:
![minvid_ubuntu_issue](https://cloud.githubusercontent.com/assets/20083658/20400567/ecf407d4-acfd-11e6-9cb0-4f21f3f63961.gif)

**Comment by 6a68** [comment]
Good catch. We probably should disable fullscreen mode for the mini player.

**Comment by 6a68** [comment]
Bug isn't observed on mac or windows. Created a linux-only label for these kinds of bugs; probably they will be fairly low on the priority list, since the number of users is really low.

**Comment by johngruen** [comment]
🙄
